From fe8eeb785677c24cba5f444bc99df290bddd0f0f Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Thu, 20 May 2021 16:39:23 +0200
Subject: [PATCH] arm64: dt: imx8mp-mba8mpxl: fix pin mux conflict

commit 0b057ec3784c ("arm64: dt: imx8mp-mba8mpxl: add remaining GPIO")
muxes also the SAI2 pins used to connect to the codec
as gpio. As a result, muxing all pins for GPIO4 fails and
in turn loading GPIO4 driver instance, too. Fix this.


Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm64/boot/dts/freescale/imx8mp-mba8mpxl.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-mba8mpxl.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-mba8mpxl.dtsi
index 28c63001a8a9..754fe87e2ca8 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-mba8mpxl.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-mba8mpxl.dtsi
@@ -835,14 +835,6 @@
 		fsl,pins = <
 			MX8MP_IOMUXC_SAI1_MCLK__GPIO4_IO20	0x180
 
-			MX8MP_IOMUXC_SAI2_RXFS__GPIO4_IO21	0x180
-			MX8MP_IOMUXC_SAI2_RXC__GPIO4_IO22	0x180
-			MX8MP_IOMUXC_SAI2_RXD0__GPIO4_IO23	0x180
-			MX8MP_IOMUXC_SAI2_TXFS__GPIO4_IO24	0x180
-			MX8MP_IOMUXC_SAI2_TXC__GPIO4_IO25	0x180
-			MX8MP_IOMUXC_SAI2_TXD0__GPIO4_IO26	0x180
-			MX8MP_IOMUXC_SAI2_MCLK__GPIO4_IO27	0x180
-
 			MX8MP_IOMUXC_SAI3_RXFS__GPIO4_IO28	0x180
 			MX8MP_IOMUXC_SAI3_RXC__GPIO4_IO29	0x180
 			MX8MP_IOMUXC_SAI3_RXD__GPIO4_IO30	0x180
-- 
2.17.1

